lib/matplotlib/backend_bases.py

```diff
@@ -2040,12 +2044,68 @@ def draw_idle(self, *args, **kwargs):
             with self._idle_draw_cntx():
                 self.draw(*args, **kwargs)
 
+    @property
+    def device_pixel_ratio(self):
+        """
+        The ratio of logical to physical pixels used for the canvas on screen.
```

Comment:
Should this be "physical to logical" instead? (ie. if I have a retina display, this value is 2 right, which is physical/logical pixels?)

Reply:
True.